ide/html.editor/src/org/netbeans/modules/html/editor/gsf/HtmlStructureItem.java

```diff
@@ -91,13 +91,13 @@ public String getSortText() {
     public String getHtml(HtmlFormatter formatter) {
         formatter.appendHtml(getName());
         if (idAttributeValue != null) {
-            formatter.appendHtml("&nbsp;<font color=808080>id=");
+            formatter.appendHtml("&nbsp;<font color=808080>#");
```

Comment:
Nitpic: should add `NOI18N` like the below.

Reply:
To Line 94? Will do. Thx.

Reply:
Yes.